.github/workflows/publish_release.yml

```diff
@@ -1,79 +1,61 @@
 name: Publish a new release
 
 on:
-  workflow_dispatch:
-    inputs:
-      is_major:
-        type: boolean
-        required: false
-        description: Make a major version
-      breaking_changes:
-        type: string
-        required: false
-        description: Breaking changes message
+  push:
+    branches:
+    - main
+    
```

Comment:
We want to use a button to trigger the action, not the main branch for the moment